packages/styles/components/_c.tooltip.scss

```diff
@@ -130,6 +160,12 @@
         left: 50%;
         transform: translate(-50%, -100%) rotate(90deg);
       }
+
+      &::before {
+        top: -1px;
+        left: 50%;
+        transform: translate(-50%, -100%) rotate(90deg);
+      }
```

Comment:
The values of the `::before` should be the same as the `::after`.
_(same remark as above)_